$Id$
excluded pselect for now, as there is no support for it in kernel-2.4
--- ltp-full-20061121.orig/testcases/kernel/syscalls/Makefile	2006-11-21 21:31:28.000000000 +0100
+++ ltp-full-20061121/testcases/kernel/syscalls/Makefile	2006-12-06 11:24:39.000000000 +0100
@@ -25,7 +25,7 @@ EXCLUDE_DIR=epoll
 #
 # Commented this out since there are directories here we don't want built by default
 #
-SUBDIR = `ls */Makefile | sed "s/Makefile//g" | grep -v epoll` 
+SUBDIR = `ls */Makefile | sed "s/Makefile//g" | grep -vE "^epoll|pselect"` 
 UCLINUX_SUBDIR = `ls */Makefile | sed "s/Makefile//g" | grep -vE "^fork|epoll|capget|capset|chmod|chown|llseek|nftw|clone|profil|getcontext"`
 
 all:
